src/services/layerHelper.js

```diff
@@ -106,12 +106,13 @@ ngeo.LayerHelper.prototype.createBasicWMSLayer = function(sourceURL,
  * as key 'capabilitiesStyles' as param of the new layer.
  * @param {string} capabilitiesURL The getCapabilities url.
  * @param {string} layerName The name of the layer.
+ * @param {string|undefined} opt_matrixSet Optional WMTS matrix set.
```

Comment:
I don't remember but the type shouldn't be `string=`?

Reply:
yes, will change